theutz/michaelutz.dev
# Reafactor Tailwind UI Landing Page

Mostyl breaking down the Tailwind UI landing page into component-sized chunks
for future manipulation

### Details

- fix(cypress,typescript): cypress typescript bug
- build(cypress): configure cypress baseUrl
- test(pages/home): site title
- chore(pages/home): remove obsolete snapshots
- test(pages/home): make sure <Head> stuff is working
- build(cypress,typescript): remove isolatedModules for cypress
- build(cypress,jest): ignore cypress tests in jest
- test(projectionist): add test and projections for component
- refactor(pages/home): extract `<Header />`
- chore(projectionist): add projections for components
- refactor(pages/home): extract `<Hero />`
- refactor(pages/home): extract `<LogoCloud />`
- refactor(pages/home): extract `<AlternatingFeatureSections />`
- refactor(pages/home): extract `<Footer />`
- build(typescript,jest): setup absolute imports
- refactor(pages/home): extract `<GradientFeatures />`
- refactor(pages/home): use absolute imports
- refactor(pages/home): make single-use components
- refactor(pages/home): extract `<Head />`
- refactor(pages/home): extract `<Stats />`
- refactor(pages/home): extract `<CTA />`
